test/raven.parsers.js

```diff
@@ -601,19 +601,17 @@ describe('raven.parsers', function () {
 
     it('should parse an error with additional information', function (done) {
       try {
-        assert.strictEqual(1, 2);
+        var err = new Error('boom');
+        err.prop = 'value';
+        throw err;
       } catch (e) {
         raven.parsers.parseError(e, {}, function (parsed) {
           parsed.should.have.property('exception');
           parsed.exception[0].stacktrace.should.have.property('frames');
           parsed.should.have.property('extra');
-          parsed.extra.should.have.property('AssertionError');
```

Comment:
It'll still work, it just makes the property name `AssertionError [ERR_ASSERTION]` instead because of some new error code stuff (for error "subtypes") in node 8. I didn't want to branch on version so I just fell back to something that checks extra property without "subtype"ing.